cnd/src/network.rs

```diff
@@ -133,26 +133,6 @@ impl Swarm {
         })
     }
 
-    /// This is the API for Alice to call in order to execute the appropriate
```

Comment:
Lol, I literally just open a PR to remove the same thing XD

Reply:
I'll drop this from this one then.